roles/idrac_bios/molecule/default/converge.yml

```diff
@@ -3,22 +3,24 @@
   hosts: all
   gather_facts: false
   tasks:
-    - name: Perform reset bios with invalid hostname
-      ansible.builtin.import_role:
-        name: idrac_bios
-      vars:
-        hostname: "randomHostname"
-        username: "{{ lookup('env', 'IDRAC_USER') }}"
-        password: "{{ lookup('env', 'IDRAC_PASSWORD') }}"
-        validate_certs: false
-        reset_bios: true
-      ignore_unreachable: true
+    ########## Below snippet is commented because of traceback error  ########
+    # - name: Perform reset bios with invalid hostname
```

Comment:
If you have raised JIT, please mention in comment if it's raising traceback error.

Reply:
Updated!